src/libs/routes/authMiddleware.ts

```diff
@@ -22,6 +22,7 @@ else {
 }
 if ( !decodedUser ) {
   throw new Error('Unauthoriesd');
```

Comment:
You have not updated authMiddleware to check the existence of the user inside DB and throw an error if the user does not exist.